.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: android
 env:
-    - BUILD_TYPE=android TERM=dumb
```

Comment:
`TERM=dumb` was added because the gradle output is crazy without it - which is fine when it's not erroring - but when it is, it's sad.